var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -33,17 +33,14 @@ class Openssl(Package):
        Layer Security (TLS) and Secure Sockets Layer (SSL) protocols.
        It is also a general-purpose cryptography library."""
     homepage = "http://www.openssl.org"
-    url      = "http://www.openssl.org/source/openssl-1.0.1h.tar.gz"
+    url = "https://www.openssl.org/source/openssl-1.0.1h.tar.gz"
```

Comment:
Can you remove this change to URL? We just finished a PR that bounced back and forth between versions and we decided on http.